dnsapi/dns_mydevil.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+# MyDevil.net API (2019-02-03)
+#
+# MyDevil.net already supports automatic Let's Encrypt certificates,
+# except for wildcard domains.
+#
+# This script depends on `devil dns` that MyDevil.net provides,
+# which means that it works only on server side.
+#
+# Author: Marcin Konicki <https://ahwayakchih.neoni.net>
+#
+########  Public functions #####################
+
+#Usage: dns_mydevil_add   _acme-challenge.www.domain.com   "XKrxpRBosdIKFzxW_CT3KLZNf6q0HG9i01zxXp5CPBs"
+dns_mydevil_add() {
+  local fulldomain=$1
```

Comment:
do not use `local` keyword.  it's bash only.

Reply:
OK. I removed them.
This script is run by bash anyway, so i'm guessing you want to keep the code "universal"? In which case, should i change also `#!/bin/bash` to `#!/bin/sh`?